samples/node/count_tokens.js

```diff
@@ -0,0 +1,316 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GoogleGenerativeAI } from "@google/generative-ai";
+import {
+  GoogleAIFileManager,
+  FileState,
+  GoogleAICacheManager,
+} from "@google/generative-ai/server";
+import fs from "fs";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const mediaPath = __dirname + "/media";
+
+async function tokensTextOnly() {
+  // [START tokens_text_only]
+  const genAI = new GoogleGenerativeAI(process.env.API_KEY);
+  const model = genAI.getGenerativeModel({
+    model: "gemini-1.5-flash",
+  });
+
+  // Count tokens in a prompt without calling text generation.
+  const countResult = await model.countTokens(
+    "The quick brown fox jumps over the lazy dog.",
+  );
+
+  console.log(countResult.totalTokens); // 11
+  console.log(countResult.contentTokens[0]);
+  /**
+   * { partTokens: [ 10 ], roleTokens: 1 }
+   */
```

Comment:
Nit: We use a variety of formats for comments that contain JSON. Some use block comments `/* */`, others use multiple single-line comments `//`. Some are formatted multi-line
```
{
  totalTokens: x
  ...
}
```
While others are inline
```
/**
 * { partTokens: x, ... }
 */

Reply:
I meant to convert them all to single line (easier to read, more compact, IDE handles it easily) - fixed this instance.